pkg/pods/manifest.go

```diff
@@ -41,6 +41,14 @@ type Manifest struct {
 	signature []byte
 }
 
+type JSONManifest struct {
```

Comment:
The problem I'm having which is the reason I created the struct in the first place is that Encode(some pods.Manifest) fails. I thought it was because Encode can't handle the map[interface{}]interface{}, though that may be incorrect.